scripts/exgfs_wave_post_gridded_sbs.sh

```diff
@@ -221,22 +222,21 @@ source "${USHgfs}/preamble.sh"
 # 1.a.2 Loop over forecast time to generate post files
 # When executed side-by-side, serial mode (cfp when run after the fcst step)
 # Contingency for RERUN=YES
-  if [ "${RERUN-NO}" = "YES" ]; then
+  if [ "${RERUN:-NO}" = "YES" ]; then
     fhr=$((FHRUN + FHMIN_WAV))
     if [ $FHMAX_HF_WAV -gt 0 ] && [ $FHOUT_HF_WAV -gt 0 ] && [ $fhr -lt $FHMAX_HF_WAV ]; then
       FHINCG=$FHOUT_HF_WAV
     else
       FHINCG=$FHOUT_WAV
     fi
-    fhr=$((fhr + FHINCG))
+    fhr=$((FORECAST_HOUR + FHINCG))
   else
-    fhr=$FHMIN_WAV
+    fhr=$FORECAST_HOUR
   fi
   fhrg=$fhr
   sleep_interval=10
   iwaitmax=120 # Maximum loop cycles for waiting until wave component output file is ready (fails after max)
-  while [ $fhr -le $FHMAX_WAV ]; do
-
+  if [ $fhr -le $FHMAX_WAV ]; then
```

Comment:
I would just take this out. It is up to the workflow to determine which hours to run for.

Reply:
@WalterKolczynski-NOAA All contents inside the loop? L239 - L432?

Reply:
Just the `if` that replaces the `while`

Reply:
Didn't delete it. Just commented it out.